spec/features/feedback_form_spec.rb

```diff
@@ -3,6 +3,12 @@
 require 'rails_helper'
 
 RSpec.describe 'Feedback Form', :js, type: :feature do
+  it 'has visibility: hidden when collapsed' do
+    visit root_path
+    feedback = page.find(:css, '#feedback', visible: false)
+    expect(feedback.native.style('visibility')).to eq('hidden')
```

Comment:
Could this just go into the other test?  It's just one feature.  Feature tests are kind of slow, so testing the before and after state in one test makes sense to me.

Reply:
I was surprised to see that there was seemingly no difference in execution time between the two, but I agree it makes sense to place them together.